sdk/core/azure-core/tests/async_tests/test_streaming_async.py

```diff
@@ -154,6 +154,7 @@ async def test_compress_plain_header(http_request):
         decoded = content.decode('utf-8')
         assert decoded == "test"
 
+@pytest.mark.live_test_only
```

Comment:
would you mind adding the same test in the async file since we skip the online one here?

Reply:
we'll do the async test later